understanding/21/text-spacing.html

```diff
@@ -18,7 +18,7 @@ <h3 id="authorresp">Author Responsibility </h3>
          <p>Further, this SC is not concerned with <em>how</em> users change the line height and spacing metrics. It does not require that content implement its own mechanisms to allow users to do this. It is not a failure of the content if a user agent or platform does not provide a way for users to do this. Content does not fail this SC if the method chosen by the user - for instance, the use of an extension or bookmarklet - fails to correctly set the line height and spacing text properties on the content (provided that the content is not actively and purposely preventing the properties from being added).</p>
          <section>
          <h4 id="applicability">Applicability</h4>
-         <p>If the markup-based technologies being used are capable of overriding text to the Success Criterion's metrics, then this SC is applicable. For instance Cascading Style Sheet/HTML technologies are quite able to allow for the specified spacing metrics. Plugin technologies would need to have a built-in ability to modify styles to the specified metrics. Currently, this SC does not apply to PDF as it is not implemented using markup.</p>
+         <p>If the markup-based technologies being used are capable of overriding text to the Success Criterion's metrics, then this SC is applicable. For instance Cascading Style Sheet/HTML technologies generally allow for the specified spacing metrics. However, if discrete portions of a markup language do not support any of the listed text style properties, then that portion of the markup language is exempt. For example, web components are a part of HTML, but do not support modification of any of the listed text style properties if they have been implemented with a closed ShadowRoot. Similarly, such web components and other plugin technologies would need to have a built-in ability to modify styles to the specified metrics. Currently, this SC does not apply to PDF as it is not implemented using markup.</p>
```

Comment:
Super picky comment (I'm happy either way):

Shouldn't web components be a proper noun (Web Components), to differentiate itself from any component-like thing on the web?

Reply:
@danhendersonede I suspect we will need to capitalize the term to reduce confusion. I'll ensure it is discussed on the AGWG call.